### ISSUE TITLE
Scanner for English version

### ISSUE THREAD

**Issue by YakivGluck** [EMCJava/WuWaOpt#7]
Seems like the've fixed text overlapping for English version. Do you plan to update the scanner tool?

**Comment by EMCJava** [comment]
Yes, I do realized that they've solved the problem, but I don't expect them to solve it by simply adding new lines🤦 (tbf it's probably the right way to do it), I'll look into it and see if it's easier to do the recognition using English OCR
